src/cli/menuItems/gpsCommands.cpp

```diff
@@ -0,0 +1,43 @@
+#include "Particle.h"
+
+#include "gpsCommands.hpp"
+#include "../conio.hpp"
+#include "../../consts.hpp"
+
+#include "gps/location_service.h"
+
+void CLI_GPS() 
+{
+    LocationPoint point = {};
+	bool run = true;
+	while(run) {
+		if(kbhit()) 
+		{
+			char ch = getch();
+			if(ch == 113) // if q is pressed
```

Comment:
Please match the whitespace convention used in other files.

Reply:
Fixed